### DISCUSSION TITLE
How to get started with contribution

### DISCUSSION THREAD

**Discussion by TacticalPrice** [wagtail/gsoc#110] [General]
Respected sir/mam
I am Ujjwal kumar , a ece undergrad , I have just entered my final year at Knit Sultanpur .I am new to open source contributions but I am well aware of python , react , postgres, c++ ,flutter, javascript and nodejs . I would love to contribute to your organization. But could you please tell me how to get started ?
Hoping to hear from you soon
Regards
Ujjwal

**Comment by thibaudcolas** [comment]
Hi @uraj323 , this project is specifically for GSoC participants. I’d suggest you look at other resources that would be more relevant to your situation.

**Reply by K-RD** [reply]
I am Robind Kumar currently persuing BTech in Data Science. I have relevent skills like Python, SQL ..., I would like to participate in GSOC@24, can you please guide me a bit.

**Reply by thibaudcolas** [reply]
https://github.com/wagtail/gsoc#2024

**Comment by iqraakhtar09** [comment]
Hello mentor @Stormheg I'm interested in Alt text capabilities, Because in one hackathon I made text to image generation app and I'm really sure that I'm able to do image to text. But I'm not be able to see Alt text capabilities GitHub kindly provide me a link and let me know I'm going to writing my proposal if there any template available for this please share it with me. When will you available to review my proposal? yeah, I'm full of excitement and potential 
here you can find me 
https://www.linkedin.com/in/iqra-akhtar-/

**Comment by vinayJ74** [comment]
hello are you particitpating in gsoc 2025

**Comment by vinayJ74** [comment]
also can you tell me from where should i start to contribute to this org
